src/teleop/jetson/arm_trajectory_server.py

```diff
@@ -30,6 +31,41 @@ def joint_states_callback(msg: JointState):
         joint_states = msg
 
 
+def euclidean_error(threshold: int, feedback: FollowJointTrajectoryFeedback) -> Tuple[bool, str]:
+    position_errors = np.array(feedback.error.positions)
```

Comment:
Would like some more description under these functions
```
"""
        Converts a Joy message with the Xbox inputs
        to a JointState to control the RA Arm with Moveit servo control
        :param msg: Has axis and buttons array for Xbox controller
        Velocities are sent in range [-1,1]
        :return:
 """
```

Like this (if you have github copilot it's pretty good at writing these)